keentools/geotracker/utils/textures.py

```diff
@@ -66,7 +71,10 @@ def frame_data_loader(index):
                 bpy_set_current_frame(frame)
             total_redraw_ui()
 
-            np_img = np_array_from_background_image(geotracker.camobj)
+            np_img = np_array_from_background_image(geotracker.camobj, index=0)
+            if np_img is None:
+                np_img = _empty_np_image()
```

Comment:
Why use empty image and not just generate error?

Reply:
We can't use errors or exceptions because it's callback function. Empty image helps to avoid interruption since it changes nothing.